### DISCUSSION TITLE
Support for converting back to regular email format

### DISCUSSION THREAD

**Discussion by marknuyens** [mckeever02/mailtolink#30] [Ideas]
First off all, thank you for that very precise tool. I find it really useful.

I could buy you a beer, but thought I'd return the favor by providing an idea/suggestion for the app itself 🎣 

Anyway, it would be nice if one could convert back an existing string to see it back in the way it was formatted in the email.

Hope this feedback helps in some way.

**Comment by NemanSyed** [comment]
Agreed. Mailto links are great until they need to be updated. If you can comfortably read the HTML you're OK, but handing the link to a business user to update can be challenging.

One of the great things about this tool is being able to give it to non-technical users to generate their own mailto links; reversing them would be beautiful.
